exporters/write_buffer.py

```diff
@@ -231,10 +230,13 @@ def get_key_from_item(self, item):
     def grouping_info(self):
         return self.items_group_files.grouping_info
 
-    def get_metadata(self, buffer_path, meta_key):
-        return self.metadata.get(buffer_path, {}).get(meta_key)
+    def get_metadata(self, key, module='write_buffer'):
```

Comment:
btw this comment is not for adding a new method, just an observation in case we need it.